snap/backend/src/utils/transactionUtils.js

```diff
@@ -90,7 +90,10 @@ const transactionUtils = {
 	createTransaction(id, meta, transaction, network, transactionTypeOverride = null) {
 		const facade = new SymbolFacade(network.networkName);
 		const senderAddress = facade.network.publicKeyToAddress(new PublicKey(transaction.signerPublicKey)).toString();
-		const date = moment.utc(facade.network.toDatetime({ timestamp: BigInt(meta.timestamp) })).format('YYYY-MM-DD HH:mm:ss');
+
+		const date = meta && meta.timestamp
```

Comment:
why would meta be undefined?

Reply:
only the timestamp will undefine as an unconfirmed transaction